data_generator.py

```diff
@@ -114,73 +116,36 @@ def time_update(rocket, time_dict):
     rocket.body_acceleration = updated_body_acceleration
 
 
-def write_data_to_file(rocket, gt_file, sd_file):
-    """
-    Writes the info of the Rocket to the ground_truth and sensor_data files.
-
-    Parameters
-    ----------
-    rocket: Rocket
-        Rocket object with info to write to file.
-    gt_file: io.TestIOWrapper
-        ground_truth file to write Rocket info to.
-    sd_file: io.TestIOWrapper
-        sensor_data file to write Rocket info to.
-    """
-    new_gt_data = [rocket.position, rocket.velocity,
-                   rocket.world_acceleration, rocket.orientation]
-
-    sensor_data = [f"{rocket.baro_pressure:.4f}", f"{rocket.temperature:.4f}",
-                   np.array2string(rocket.body_acceleration, precision=4,
-                                   floatmode='fixed'),
-                   np.array2string(rocket.body_mag_field, precision=4,
-                                   floatmode='fixed')]
-
-    gt_data = [np.array2string(e, precision=4, floatmode='fixed') for e in
-               new_gt_data]
-    gt_data_to_write = ' '.join(["{0: <33}".format(data) for data in gt_data])
-    sensor_data_to_write = ' '.join(
-        ["{0: <33}".format(data) for data in sensor_data])
-    gt_file.write(gt_data_to_write + "\n")
-    sd_file.write(sensor_data_to_write + "\n")
-
-
-# TODO: Check if the title output can be shortened
 def main():
     """
     Main function for generating data based on the input Rocket.
     """
     # Timestep setup
     time_dict = {"current_time": 0, "end_time": 100, "timestep": 0.01}
```

Comment:
Is this dictionary necessary? It seems like end_time and timestep are constants. Could all 3 of these values be stored in primitive variables?